content/docs/command-reference/get.md

```diff
@@ -219,3 +232,15 @@ Untracked files:
 [get started example repo]: https://github.com/iterative/example-get-started
 [switching between versions]:
   /doc/start/data-management/data-versioning#switching-between-versions
+
+## Example: set AWS profile for default remote
+
+```cli
+$ dvc get URL path --remote-config profile=myprofile
+```
+
+## Example: set default remote
+
+```cli
+$ dvc get URL path --remote myremote
+```
```

Comment:
Can we setup some use case to explain how they would be used?

Also it's minor, but for style consistency, I think we always use real examples here instead of generic stuff like `URL`, `path`, etc.

Reply:
@dberenbaum Good points, will do. Thanks.